pylearn2/devtools/tests/test_format.py

```diff
@@ -586,22 +586,18 @@
     'datasets/tests/test_npy_npz.py',
     'linear/tests/test_conv2d.py',
     'devtools/tests/pep8/pep8.py',
-    'devtools/tests/pep8/__init__.py']
-
-# add files which have long execution time to whitelist_docstrings
-whitelist_docstrings.extend([
-    'sandbox/cuda_convnet/debug.py',
-    'energy_functions/tests/test_rbm_energy.py',
+    'devtools/tests/pep8/__init__.py'
+    'scripts/lcc_tangents/make_dataset.py',
```

Comment:
Those two lines without a comma between them are still here (line 589). Can you check what your local version is?
Do you have outstanding local changes that you did not commit?